core/modules/query/ValueExpr.h

```diff
@@ -62,8 +62,12 @@ class ValueFactor;
 /// operators. No nesting is allowed yet.
 class ValueExpr {
 public:
-    ValueExpr();
-    enum Op {NONE=200, UNKNOWN, PLUS, MINUS, MULTIPLY, DIVIDE};
+    // in Op: DIVIDE is the `/` operator, "Division; quotient of operands" as specified by MySQL.
+    //        DIV is the `DIV` operator, "Division; integer quotient of operands" as specified by MySQL.
+    //        THE BIT_ values are bitwise operators: BIT_SHIFT_LEFT is <<, BIT_SHIFT_RIGHT is >>,
+    //        BIT_AND is &, BIT_OR is |, BIT_XOR is ^.
+    enum Op {NONE=200, UNKNOWN, PLUS, MINUS, MULTIPLY, DIVIDE, DIV, MOD, MODULO,
```

Comment:
Why does it need to start with `200`? Just curious.

Reply:
I don't know. It was that way before I started working on it. It was checked in by Daniel years ago, I can't find an explanation for it. I didn't want to change it without need.

Reply:
Makes a sense.